### ISSUE TITLE
 add "load save" (lobby) parameter

### ISSUE THREAD

**Issue by AlexTriada** [A3Antistasi/A3-Antistasi#57]

**Comment by Barbolani77** [comment]
Will study the possibility. It seems a good idea

**Comment by AlexTriada** [comment]
belive me it’s not low priority task, it’s very very important task you are very good man if you do, it’s not so complicated

**Comment by Barbolani77** [comment]
but there are more important tasks and my time is not unlimited

**Comment by AlexTriada** [comment]
yeah, I did not expect anything else, thanks

**Comment by Barbolani77** [comment]
If you want to make it quicker you can allways provide the code here and I Will implement it in seconds. 

And if not, after telling you that I Will do, you come here with hurry and get dissapointed because I tell you the priorities of the mission development?????

May I say that I am not your fucking slave or is it incorrect?

**Comment by zalexki** [comment]
@Barbolani77 actually yes it is, i understand the frustation but lets keep it respectfull and enjoyable for everyone, especially that I think you guys just misunderstood due to language translation.

In the meantime @AlexTriada you are more than welcome to fork this repo, fix this issue and send us a pull request ;)

I invite everyone to take 5minute to read this : https://symfony.com/doc/current/contributing/community/review-comments.html

**Comment by Barbolani77** [comment]
What is not subject to translations is your -1 reaction and your closing of the Issue

**Comment by Sparker95** [comment]
It's actually a functionality which we have already implemented in the community version of Antistasi. It worked this way:
There was a parameter "Allow to start a new campaign" which was OFF by default. If you wanted to start a new campaign, you'd set it to ON manually and then the menu would ask you if you wanted to make a new start.
The reason behind this is to ease the administration of automatic server restarts when there is no administration instantly available. It also helped us a lot at the official server because users would just join the game and the progress would get loaded. I am not sure if the autoload is currently present in the mission.

**Comment by zalexki** [comment]
This dude is the owner of this mission and got all rights on this code, if it says it's interresting but got more important issues that's how it is.

You've been welcomed multiple times to `quickly and very easily fix this very important issue` as you said.

Your ambiguous comments (that you deleted...) and you closing the issue show your lack of maturity.
Even worse, you are threatening people on a collaborative platform ? 

Closing issue drama has not is place here.

**Comment by AlexTriada** [comment]
This dude is the owner of this mission and got all rights on this code, if it says it's interresting but got more important issues that's how it is. - это ты вообще к чему написал?
 You've been welcomed multiple times to quickly and very easily fix this very important issue as you said.
- и что, это даёт возможность мне хамить???
Your ambiguous comments (that you deleted...) and you closing the issue show your lack of maturity.
Even worse, you are threatening people on a collaborative platform ? - это вообще враньё,, откуда ты это взял?
Closing issue drama has not is place here. - это новое слово в драме? не слыхал о таком - я автор темы, хочу - закрываю

**Comment by AlexTriada** [comment]
всё что ты написал выше это даже не признак незрелости, это глупость и инфантильность

**Comment by zalexki** [comment]
@AlexTriada I repeat you are more than welcome to help posting issues and PR on this repository, in english ;)

Issue will be treated later on.

**Comment by Barbolani77** [comment]
Thanks @zalexki. Back to the roots, omitting his lack of social skills and education, Alex idea was good and Will be implemented (once I finish the whole AI suite which is by far more important).

**Comment by Barbolani77** [comment]
Will try to persistent save YES by default and only being able to be changed by server admins, so new starts Will allways depend on an admin.


Switch commander yes by default and membership yes by default, but those will get overriden once the load is done.

**Comment by zalexki** [comment]
Can you check @Sparker95 solution that was on 1.8 version ?
The goal is to actually not need any admin at mission restart, in cohesion with the member server/mission restart, especially in the case of auto restart servers.

**Comment by Barbolani77** [comment]
its more or less the same

I Will add this and some other options with defaults.

Life will be easier.

**Comment by Sparker95** [comment]
I proposed a similar solution in a forum thread: so for each of theese options we could make a parameter in the parameters menu. Each parameter will have three options: **Force ON, Force OFF, Default**.
The 3rd one (Default) will cause default behaviour with GUI appearing at mission start. Others will force the corresponding options to one of the two states.
So We have total compatibility with people already running the mission and we add possibility for other admins to set it to whatever they want.

**Comment by Barbolani77** [comment]
These are my notes of the params I Will add, pls do not hesitate to suggest a few more:

Load last Save: def Yes
Server membership (Overriden upon Load): def Yes.
Switch Comm: Def yes.
TK Punish: Def Yes.
Mission Radius: Def 4Kmts (4,8,12)
Allow PvP: def No
Allow player markers: def Yes
AI Skill: def Medium (mult 0.5,1,2)
No of same ítems in Arsenal to unlock: def 25 (15,25,50)
Civ Traffic Level: def Medium (mult 0.5,1,2)

So if the server is started by an admin he Will be able to tweak whatever, if not, the default values are applied, and those are suitable for open dedis, no conflict with anything.

From there, JiP players Will see the traditional Load window for their personal saves, and nothing else.

**Comment by Sparker95** [comment]
Arsenal restrictions for non-members: allow/forbit takingg non-unlocked weapons.
In case people want to have membership without these weapon restrictions.

**Comment by Barbolani77** [comment]
Non members picking non unlocked weapons can mess the game, as in other topics, so if ppl wants to allow that kind of thing they may just disable the membership requirement.

Closing as it's already implemented.
